app/constants/Strings.ts

```diff
@@ -62,5 +62,8 @@ export default {
   ios: 'IOS',
   permissions: 'PERMISSIONS',
   weNeedPermission: 'you need to provide permissions to use this feature!',
-  providePermission: 'Provide Permission'
+  providePermission: 'Provide Permission',
+  videosEndpoint: 'https://gist.githubusercontent.com/deepakpk009/99fd994da714996b296f11c3c371d5ee/raw/28c4094ae48892efb71d5122c1fd72904088439b/media.json',
+  http: 'http',
+  https: 'https'
```

Comment:
why do you require this string file ?

Reply:
because the videos url starts with http that are not supported by iPhone so we have to replace HTTP with https. 
 **<Image
        source={{uri: video.thumb.replace(Strings.http, Strings.https)}}
        style={styles.videoThumbNail}
      />**
      
check this piece of code in VideoComponent.tsx

Reply:
There is a way in iOS that will allow HTTP URL accessible 
NOTE:- we can't use it in production but for you try to explore more then comment back here again

Reply:
can you please guide me regarding this in detail how to configure it!
I tried **<key>NSExceptionDomains</key>
  <dict>
    <key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
    <true/>
  </dict>
</dict>**

but does not work && and can we skip this for now because allowing arbitrary HTTP requests globally is less secure than specifically turning these two HTTP endpoints to https **as it gives more control!!**